### ISSUE TITLE
Question about training settings for warm-up phase

### ISSUE THREAD

**Issue by yyj828** [buggyyang/CDC_compression#24]
Hi, yang！Thank you for your great work! I would like to inquire about the training command used for the warm-up phase. Are all settings identical to those in xparam/train.sh, except for 𝜆 = 0.9, the training step = 700000, as mentioned in the paper?
Thank you for your assistance!

**Comment by buggyyang** [comment]
Sorry for the late response. Basically for the warm-up phase, the only thing you need to train a model with very high bitrate. Then the followup training should be fairly smooth by following the paper.
